string soma_roi_id
string type
int32 roi_knowledge
#HourlyKnowledgeMsg[] roi_knowledge_hourly
